### PR TITLE
chore: sync common files

### PR DESCRIPTION
**Pull request by mcdurdin**
This synchronises the new `MarkdownHost` class between keymanapp/keyman.com#192 and help.keyman.com.

It adds also `ParsedownExtra` support so we can do definition lists, which I have validated with the `contextExOutput` file.

### CODE REVIEW

File: _includes/md/mdhost.php [hunk 1]
```diff
@@ -1,57 +1,16 @@
 <?php
   require_once('includes/template.php');
```
Comment (darcywong00) [inline _includes/md/mdhost.php:2]: The keyman.com version also has this:
```
require_once __DIR__ . '/../../autoload.php';
```

**Review by darcywong00 — APPROVED** [review]
lgtm  (is autoload not needed?)

File: _includes/md/mdhost.php (hunk 1 above)
Comment (mcdurdin) [reply _includes/md/mdhost.php:2]: Yes, on help.keyman.com, the autoload is included automatically, but we haven't yet done the work on keyman.com for that.

**Review by darcywong00 — CHANGES_REQUESTED** [review]
oops, I noticed the check fails from PHP errors referencing Parsedown.php
history_utils.php
knowledge-base/index.php

**Review by darcywong00 — APPROVED** [review]
lgtm

**Pull request merged**